Add instruction for changing Composer version (#8893)
magento/devdocs: src/cloud/docker/docker-config.md

echo "127.0.0.1 magento2.test" | sudo tee -a /etc/hosts
```

## Change Composer version

> You can change Composer version by adding next line in `.docker/config.env` configuration file:

Reviewer: ```suggestion
You can change Composer version by adding the following line to the `.docker/config.env` configuration file:
```
